releases/mainnet/1.2.10/index.html

```diff
@@ -0,0 +1,24 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.2.10</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.2.10</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.2.10/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmQBFYzikwVfwH1nE92eKtjX9xRFjTRG8rqbWBAg2STG9k">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.2.10/abi/index.html">ABIs</a></li>
+				 <li>Links to GitHub repositories</li>
+					<ul>
+						DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.2.10" target="_blank">changelog</a>)</li>
```

Comment:
Release notes are not ready yet. Either comment out or wait for dss-deploy-scripts release to be added to GitHub.

Reply:
I've commented this out.